pkg/controller/application/application_controller_test.go

```diff
@@ -891,8 +891,8 @@ func (f *fixture) newController() (*Controller, shipperinformers.SharedInformerF
 
 	const noResyncPeriod time.Duration = 0
 	shipperInformerFactory := shipperinformers.NewSharedInformerFactory(f.client, noResyncPeriod)
-
-	c := NewController(f.client, shipperInformerFactory, f.resolveChartVersion, record.NewFakeRecorder(42))
+	var resync time.Duration = 1
```

Comment:
Please consider using constants from time package, say, time.Second instead of simple casted integer: it provides time-duration semantics to the reader.